lms/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: lms
 description: Helm chart for MOLT Live Migration Service
 type: application
-version: 0.2.5
-appVersion: 0.2.4
+version: 0.2.7
```

Comment:
Why we advance from 0.2.5 to 0.2.7, but not 0.2.6?

Reply:
I forgot to update to 0.2.6 last time. I'm correcting it now.